ej2-react/schedule/how-to/set-different-event-time-duration.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
-title: Set different event time duration in React Schedule component | Syncfusion
-description: Learn here all about Set different event time duration in Syncfusion React Schedule component of Syncfusion Essential JS 2 and more.
+title: Set Time duration | React Schedule Component | Syncfusion
```

Comment:
Revert this

Reply:
Comment resolved @gowthamanthavasiyappan